Source/WebKit/UIProcess/Extensions/Cocoa/API/WebExtensionContextAPIAlarmsCocoa.mm

```diff
@@ -44,7 +44,7 @@
 
 void WebExtensionContext::alarmsCreate(const String& name, Seconds initialInterval, Seconds repeatInterval)
 {
-    m_alarmMap.set(name, WebExtensionAlarm::create(name, initialInterval, repeatInterval, [&](const WebExtensionAlarm& alarm) {
+    m_alarmMap.set(name, WebExtensionAlarm::create(name, initialInterval, repeatInterval, [this, protectedThis = Ref { *this }](const WebExtensionAlarm& alarm) {
```

Comment:
What's the difference between `this` and `protectedThis`?

Reply:
`this` is implicitly used by the member function call in the block. The `protectedThis` holds a reference to keep `this` alive, since it won't be retained otherwise.